apps/nextjs-website/src/lib/api.ts

```diff
@@ -49,19 +48,7 @@ export const getFooterProps = async (): Promise<FooterProps> => {
 };
 
 // Return PageProps for a specific page ID
-export const getPageProps = async (
-  path: string
-): Promise<PageData['data']['attributes'] | null> => {
-  const allPages = await getAllPages(); // Fetches directly from NextJS internal cache (will not query server each time)
-
-  const pageID = allPages.filter((page) => path === page.slug.join('/'))[0]?.id;
-
-  if (!pageID) {
-    return null;
-  }
-
-  const {
-    data: { attributes },
-  } = await getPage(pageID, appEnv);
-  return attributes;
+export const getPageProps = async (path: string): Promise<Page | undefined> => {
+  const allPages = await getAllPages();
+  return allPages.find((page) => path === page.slug.join('/'));
```

Comment:
I'd change the input from `string` to `ReadonlyArray<string>`, in this way the [caller](https://github.com/pagopa/b2b-portals/pull/82/files#diff-d853063839a7021acab37ca1300f582473ff4136295437567bab45b59ad40695R17) does not have to transform the argument

```suggestion
export const getPageProps = async (slug: ReadonlyArray<string>): Promise<Page | undefined> => {
  const allPages = await getAllPages();
  return allPages.find((page) => slug.join('/') === page.slug.join('/'));
```